.emacs.d/test/lisp/dot-tests.el

```diff
@@ -0,0 +1,27 @@
+(require 'buttercup)
+
+(defun kotct/load-corresponding ()
+  (when noninteractive
+    (assert (string-match-p "-test\\.el$" load-file-name))
+    (load-file
+     (replace-regexp-in-string
+      "-test\\.el$" ".el"
+      (replace-regexp-in-string
+       (regexp-quote "/test/") "/"
+       load-file-name)))))
+
+(defun kotct/run-tests ()
+  (setf buttercup-suites nil)
+  (load "~/.emacs.d/test/lisp/behavior/completion-c-test.el" nil t)
```

Comment:
`nil t` is necessary, otherwise we get a bunch of messages saying it loaded all the test files. It's annoying. I'm making this line dynamic so we don't have to specify all the test files.